### PR TITLE
Use Datadog's standard attributes

### PR DESCRIPTION
**Pull request by Supy**

### CODE REVIEW

File: lib/lograge/log_subscribers/action_controller.rb [hunk 1]
```diff
@@ -19,13 +19,34 @@ def unpermitted_parameters(event)
...
 
       def initial_data(payload)
-        {
-          method: payload[:method],
-          path: extract_path(payload),
+        initial_data = {
           format: extract_format(payload),
-          controller: payload[:controller],
-          action: payload[:action]
+          http: {
```
Comment (oliver-hohn) [inline lib/lograge/log_subscribers/action_controller.rb:37]: Should we include the `path` as well? If we make it a facet then we will be able to, for example, look at all requests to an endpoint across all environments.

File: lib/lograge/log_subscribers/base.rb [hunk 2]
```diff
@@ -27,11 +27,11 @@ def process_main_event(event)
 
       def extract_request(event, payload)
         data = initial_data(payload)
-        data.merge!(extract_status(payload))
+        data.deep_merge!(extract_status(payload))
         data.merge!(extract_runtimes(event, payload))
         data.merge!(extract_location)
         data.merge!(extract_unpermitted_params)
```
Comment (oliver-hohn) [inline lib/lograge/log_subscribers/base.rb]: For consistency, should we make these also be deep merges?

File: lib/lograge/log_subscribers/action_controller.rb (hunk 1 above)
Comment (Supy) [reply lib/lograge/log_subscribers/action_controller.rb:37]: Path will automatically be extracted by Datadog from the URL

File: lib/lograge/log_subscribers/action_cable.rb [hunk 3]
```diff
@@ -27,7 +25,7 @@ def default_status
       end
 
       def extract_runtimes(event, _payload)
-        { duration: event.duration.to_f.round(2) }
+        { duration: 1_000 * event.duration }
```
Comment (oliver-hohn) [inline lib/lograge/log_subscribers/action_cable.rb:30]: Is this changing the `duration` to be in nanoseconds (ref: https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/#performance)? If so, could you add a comment above it?

File: spec/log_subscribers/action_cable_spec.rb [hunk 4]
```diff
@@ -28,144 +29,146 @@
...
       subscriber.perform_action(event)
-      expect(log_output.string).to be_present
+      expect(log_output).to_not be_empty
     end
   end
 
   context 'when processing an action with lograge output' do
-    before do
-      Lograge.formatter = Lograge::Formatters::KeyValue.new
+    it 'includes the controller and action' do
```
Comment (oliver-hohn) [inline spec/log_subscribers/action_cable_spec.rb]:
```suggestion
    it 'includes the controller' do
```

**Review by oliver-hohn — APPROVED** [review]

File: lib/lograge/log_subscribers/action_cable.rb (hunk 3 above)
Comment (Supy) [reply lib/lograge/log_subscribers/action_cable.rb:30]: Yessir. Will add a comment.

**Pull request merged**